src/Rule/Nested.php

```diff
@@ -167,7 +171,11 @@ private static function ensureArrayHasRules(iterable &$rules)
                 continue;
             }
             if (!$rule instanceof RuleInterface) {
-                $message = sprintf('Each rule should be an instance of %s, %s given.', RuleInterface::class, get_debug_type($rule));
```

Comment:
Ideally this should be handled by a linter - provide a warning about exceeding max line length or split arguments automatically.

Reply:
https://github.com/yiisoft/docs/issues/175